README.md

```diff
@@ -1,28 +1,31 @@
-# minimal-gltf-loader
+# minimal-gltf-loader-typescript
```

Comment:
Why is all the screen shot, live demo link, javascript descriptions get removed?
The js lib and description should still be there. Typescript version is not directly usable but need a build process, which might not fit needs for all developers. We should have another new section in the description to tell there's a new ts version.